notebooks/quickstart/data-owner/03-join-network.ipynb

```diff
@@ -0,0 +1,359 @@
+{
```

Comment:
<p>Can we also define in text before this cell that the default admin credentials to the domain are:</p><ul><li>Email: <code>info@opemined.org</code></li><li>Password: <code>changethis</code></li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/6693/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
It is there already just above the cell, "The below cell has default credentials, please change accordingly."
Let me know if you meant rephrasing/changing it?